opening_hours/utils/hauki_api_client.py

```diff
@@ -14,18 +21,84 @@
 
 
 class HaukiAPIClient:
-    """Hauki API client, minimal functionality implemented."""
-
-    @staticmethod
-    def build_url(endpoint: Literal["resource", "opening_hours", "date_period"], resource_id: str | None = None) -> str:
+    def __init__(self):
         if not settings.HAUKI_API_URL:
             raise HaukiConfigurationError("HAUKI_API_URL environment variable must to be configured.")
 
-        url = urljoin(settings.HAUKI_API_URL, f"/v1/{endpoint}/")
-        if resource_id:
-            url += f"{resource_id}/"
+    ############
+    # resource #
+    ############
+
+    def get_resources(self, *, hauki_resource_ids: list[int], **kwargs: Any) -> HaukiAPIResourceListResponse:
+        # Prepare the URL
+        url = urljoin(settings.HAUKI_API_URL, "/v1/resource/")
+        query_params = {
+            "resource_ids": ",".join(str(id_) for id_ in hauki_resource_ids),
+            **kwargs,
+        }
+
+        # Get the data from Hauki API
+        response: HaukiAPIResourceListResponse = self.get(url=url, params=query_params)
```

Comment:
Unnecessary assignment before return (I think this is a [`ruff` lint rule](https://docs.astral.sh/ruff/rules/unnecessary-assign/), should we enable it?)